src/main/thrift/parquet.thrift

```diff
@@ -572,14 +572,14 @@ union BloomFilterAlgorithm {
 /** Hash strategy type annotation. It uses Murmur3Hash_x64_128 from the original SMHasher
  * repo by Austin Appleby.
  **/
-struct Murmur3 {}
+struct Murmur3Hash {}
 /** 
  * The hash function used in Bloom filter. This function takes the hash of a column value
  * using plain encoding.
  **/
 union BloomFilterHash {
   /** Murmur3 Hash Strategy. **/
-  1: Murmur3 MURMUR3;
+  1: Murmur3Hash MURMUR3HASH;
```

Comment:
```suggestion
  1: Murmur3Hash MURMUR3;
```

The current naming convention allows less verbose names for the members of the union, it is enough if the global struct is verbose.

Reply:
I see. Thanks.